extensions/security-jpa/deployment/src/test/java/io/quarkus/security/jpa/FormAuthJpaTestCase.java

```diff
@@ -0,0 +1,96 @@
+package io.quarkus.security.jpa;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+import java.util.function.Supplier;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+import io.restassured.filter.cookie.CookieFilter;
+
+public class FormAuthJpaTestCase {
+
+    private static final String APP_PROPS = "" +
+            "quarkus.datasource.db-kind=h2\n" +
+            "quarkus.datasource.username=sa\n" +
+            "quarkus.datasource.password=sa\n" +
+            "quarkus.datasource.jdbc.url=jdbc:h2:mem:minimal-config'\n" +
+            "quarkus.hibernate-orm.sql-load-script=import.sql\n" +
+            "quarkus.hibernate-orm.database.generation=drop-and-create\n" +
+            "#quarkus.hibernate-orm.log.sql=true\n" +
+            "quarkus.http.auth.form.enabled=true\n" +
```

Comment:
Are these all really required? https://quarkus.io/guides/security#form-based-authentication says they all have a default, except the encryption key, but IMO even that one could default to one generated at startup.

Reply:
I just copied from the existing test, you don't actually need all of them.

Reply:
Hah, OK.